helm/kong-app/UPGRADE.md

```diff
@@ -56,6 +57,43 @@ text ending with `field is immutable`. This is typically due to a bug with the
 If you encounter this error, deleting any existing `init-migrations` jobs will
 clear it.
 
+## 2.0.0-rc1
+
+### Support for Helm 2 dropped
```

Comment:
`apiVersion` in the `Chart.yaml` file still says `v1`. I'm unsure if we should bump the `apiVersion` as well.

Reply:
@ubergesundheit I see what you mean. That was my oversight. I'll fix it right away.